maritime_interfaces/support/shared/ComponentInfo.msg

```diff
@@ -0,0 +1,9 @@
+# ComponentInfo.msg
```

Comment:
Does this exist as a CAN message already?

Reply:
It does, but doesn't mean we don't want something on the other side of the integration service.